### PR TITLE
create balena.yml for open fleets

### PR DESCRIPTION
**Pull request by shawaj**
create balena.yml for open fleets

@mpous - does this look like it will work?

cc @ryanteck

### CODE REVIEW

File: balena.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    data:
+      url: "https://cdn.shopify.com/s/files/1/0071/2281/3001/files/Nebra-Icon-Blue_bb758eaa-b10d-4ea6-a55e-babd69135b69.png?v=1620938774"
+  defaultDeviceType: raspberry-pi
+  supportedDeviceTypes:
+    - raspberry-pi
+    - raspberry-pi2
+    - raspberrypi3
+    - raspberrypi3-64
+    - raspberrypi4-64
+    - raspberrypi400-64
```
Comment (ryanteck) [inline balena.yml]: I think I'm going to remove 400 as don't really think it's one to support.

File: balena.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    - raspberrypicm4-ioboard
+    - fincm3
```
Comment (ryanteck) [inline balena.yml:29]: Think we'll add in Beaglebone Black, possibly can add in the Odroid C1+ along with the Asus Tinkerboard for lights if you would like.

**Review by ryanteck — COMMENTED** [review]
Just a few minor changes, would you like to do or me to accept PR then I change them?

File: balena.yml (hunk 1 above)
Comment (shawaj) [reply balena.yml]: yeah probably not particularly relevant - removed

File: balena.yml (hunk 2 above)
Comment (shawaj) [reply balena.yml:29]: added in all the beaglebones, odroid and tinker.

could add in banana pi m1+ and orange pi one potentially? although not sure the 40pin header pinout is the same

**Pull request merged**